client/src/components/Slider/Slider.css

```diff
@@ -0,0 +1,48 @@
+.slide {
+  height: 280px;
+  width: 260px;
+  margin-right: 30px;
+}
+
+.title {
+  width: 290px;
+  background-color: rgb(246, 230, 222);
+  text-align: left;
+  font-weight: bold;
+}
+
+.card {
+  display: flex;
+}
+
+.slide-container {
+  display: flex;
+  flex-direction: row;
+  height: 260px;
+  width: 300px;
+  background-color: rgb(246, 230, 222);
+  margin-right: 0;
+}
+
+.button {
+  background-color: rgb(246, 230, 222);
+  border: 0;
+  margin: 0;
+  font-size: 20px;
+  height: 278px;
+}
+
+.slide-img {
```

Comment:
I would update the spacing between the images here to more closely resemble the figma.

Reply:
I did my best to get closer to the look of the mockup.  Please let me know if you would like me to fine tune it a bit more, or if I should try and incorporate the exact pixel spec of the mockup.